### ISSUE TITLE
Rename repository

### ISSUE THREAD

**Issue by trevorbryant** [opencontrol/certifications#8]
In the Risk Management Framework and FedRAMP, there is no concept of certifications. Instead, this terminology has been replaced with "authorization". 

Strongly recommend renaming the mentions of "certification" to "authorization" to stay consistent with the US Government's vocabulary.

**Comment by shawndwells** [comment]
💯  +1

**Comment by shawndwells** [comment]
This will totally break many downstream systems. This could be swapped in a major version bump of the schema though.

**Comment by trevorbryant** [comment]
I understand. That's difficult.

Is there any way to track back to see just what it impacts?

Edit: I removed a statement as it seemed harsh when it was not intended to be.

**Comment by shawndwells** [comment]
Not really. It's part of the schema, so "whoever uses the schema" would be impacted.

For example, at Red Hat, that would be the entire backend of http://atopathways.redhatgov.io/

**Comment by shawndwells** [comment]
Specific example - https://atopathways.redhatgov.io/ato/products/openshift-container-platform-4/NIST-800-53

![image](https://user-images.githubusercontent.com/5713754/112369484-9bbf3980-8cb2-11eb-9b3d-337b93792d20.png)
